server/app.py

```diff
@@ -111,18 +111,50 @@ def fetchArtInformation(i):
     return cardSet
 
 
+def selectArt (selectedArtist):
+
+    artArray = [[438821,436449,437999,438001,436448,337858,344577,337172],
+    [336327,436527,436530,436531,436524,436534,437998,436529],
+    [437397,437392,337491,437390,437402,437396,437389,437398],
+    [10793,10786,10795,10796,10794,10790,10788,10798],
+    [10154,10155,10151,10150,10158,21186,10156,10149],
+    [435882,435885,437990,435871,435874,435873,435870,435879],
+    [786115,370826,365307,337070,786093,785991,785974,691010],
+    [752047,11865,11859,11862,11866,11860,11863,11872]]
+    
+    artists= ["Paul Gauguin", "Vincent van Gogh", "Rembrandt", "Asher Brown Durand", "Albert Bierstadt", "Paul Cézanne", "Auguste Edouart", "Frederic Remington"]
+    
+    for a in range(8):
```

Comment:
Instead of iterating through the array, maybe you could use python's index function to your advantage?
`artistIdx = artists.index(selectedArtist)`

Reply:
Great suggestion! Wish I had thought of that - I've implemented it!

Reply:
Now that I am testing, I remember that we do need to iterate through the array to ensure that the selected artist does in fact exist in the array. I thought the index function would return a -1 value for example, but it throws an error so I am not sure it is the best implementation